.gitignore

```diff
@@ -78,6 +78,8 @@ datasets/*
 !datasets/karate-disjoint.csv
 !datasets/netscience.csv
 
+# nx-python side effects
```

Comment:
Was this supposed to be `# nx-cugraph side effects`?

Reply:
haha, yes.  oops.  fixed.